src/entities/protocols/unwrapWETH.ts

```diff
@@ -13,9 +13,9 @@ export class UnwrapWETH implements Command {
   readonly routerAddress: string
   readonly amount: BigNumberish
 
-  constructor(amount: BigNumberish, chainId: number, permit2?: Permit2Permit) {
+  constructor(amount: BigNumberish, chainId: number, permit2?: Permit2Permit, routerAddress?: string) {
```

Comment:
per this PR, we no longer need this here. can do ROUTER_AS_RECIPIENT  on line 30.

https://github.com/Uniswap/universal-router/pull/205/files

Reply:
Thats actually sadly not true for _this_ PR because the SDK is still building for the deployed version of UR - and that PR you've linked was after we deployed UR 😭 

However this is a good spot - because I will make this change in #112 which is where i'm updating the SDK for our new UR version

Reply:
ohhh dang. I guess it's purely for testing reasons so users don't need to know about it, so that's a +-ish....